code/parachain/frame/composable-traits/src/currency.rs

```diff
@@ -21,6 +21,12 @@ pub trait CurrencyFactory<AssetId, Balance> {
 	fn reserve_lp_token_id(ed: Balance) -> Result<AssetId, DispatchError> {
 		Self::create(RangeId::LP_TOKENS, ed)
 	}
+	/// Given a local `AssetId` (within the range of `0` to `u32::MAX`) returns a global (within the
+	/// scope of Currency Factory) `AssetId`.
+	fn local_to_global_asset_id(
```

Comment:
This naming is actually confusing as we use "local asset" to refer to asset IDs that are native to Picasso chain as opposed to foreign assets that are XCM transferred or bridged. We need a better name for this encoding. It's essentially a numeric system with `u32::MAX` as the base. May be we can call it `u32_base_asset_id`

Reply:
scoped_
protocol_

 I prefer protocol_

Reply:
06b3cc9bca0db927b0a6aa6cfcf3cc0b0330e876

Reply:
How's this? @vimukthi-git @dzmitry-lahoda

Reply:
I personally li8ke `protocol_`, since it's going from protocol -> global. `base_u32_` is fine too though

Reply:
a87d8b2c974f37872cca55531e9ae89f68783de3
Sticking with `protocol_asset_id_to_unique_asset_id`